scripts/generators/patternfly-4-component/templates/component/{{componentName}}.js

```diff
@@ -1,12 +1,25 @@
 import React from 'react';
+import styles from '@patternfly/patternfly-next/{{typeDir}}/{{componentName}}/styles.css';
+import { css } from '@patternfly/react-styles';
+import PropTypes from 'prop-types';
 
-const propTypes = {};
+const propTypes = {
+  children: PropTypes.node,
```

Comment:
You might check this works with strings. It is common to just throw a string in there instead of a div/span